client/me/purchases/cancel-purchase/loading-placeholder.jsx

```diff
@@ -1,33 +1,94 @@
-import { Button, Card, CompactCard } from '@automattic/components';
-import { localize } from 'i18n-calypso';
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate, localize } from 'i18n-calypso';
+import page from 'page';
 import PropTypes from 'prop-types';
-import LoadingPlaceholder from 'calypso/me/purchases/components/loading-placeholder';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import { CALYPSO_CONTACT } from 'calypso/lib/url/support';
 import titles from 'calypso/me/purchases/titles';
 
+import '../components/loading-placeholder/style.scss';
+
 const CancelPurchaseLoadingPlaceholder = ( { purchaseId, siteSlug, getManagePurchaseUrlFor } ) => {
+	const translate = useTranslate();
+	const title = titles.cancelPurchase;
+	const isFullWidth = true;
 	let path;
 
 	if ( siteSlug ) {
 		path = getManagePurchaseUrlFor( siteSlug, purchaseId );
 	}
 
+	function goBack() {
```

Comment:
I believe what @sirbrillig [commented previously](https://github.com/Automattic/wp-calypso/pull/69276#discussion_r1004638963) also applies here: the actual definition of`goBack()`, `supportLink()` and `siteHeader()` will change everytime `CancelPurchaseLoadingPlaceholder` is called. In general, if a lambda function won't change based on the runtime context of the function it resides in, most of the time it'd better to be extracted. The end result also tends to be more readable.

There are some other similar cases throughout this PR that worth considering this. I'll defer that to you :)

Reply:
Fixed, I'll avoid reusing functions like this

Reply:
Fixed, I'll avoid to use functions in this way